### PR TITLE
Command Age Requirements

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request

This PR makes it so that lesser command roles require a character be at least 25, while higher standards command roles require they be at least 28.

## Why It's Good For The Game

Making characters actually have an expectation that, in-world, they'd had more than a high-school education, will lead to more confidence.

Additionally, the extra backstory planning for characters that are older will lead to higher quality characters.
## Proof Of Testing
It's joever
image

## Changelog

:cl: ReturnToZender
add: Command character age requirements are back in town!
/:cl:

### CODE REVIEW

**Comment by Tupinambis** [comment]
No? The confidence issue is entirely OOC, nobody ever looks at ages except to be annoying about it. 
Secondly, what backstory planning? Stronger characters? It's a number you can just make go up or down it's 0 effort. If this change goes through I will just increase my characters age and change nothing else because character age has never been relevant to how I RP. She's old enough to have graduated, good enough for me tbqh.

**Comment by ReturnToZender** [comment]
> No? The confidence issue is entirely OOC, nobody ever looks at ages except to be annoying about it. Secondly, what backstory planning? Stronger characters? It's a number you can just make go up or down it's 0 effort. If this change goes through I will just increase my characters age and change nothing else because character age has never been relevant to how I RP. She's old enough to have graduated, good enough for me tbqh.

Not a problem to adjust your age to match

**Comment by Odairu** [comment]
> > No? The confidence issue is entirely OOC, nobody ever looks at ages except to be annoying about it. Secondly, what backstory planning? Stronger characters? It's a number you can just make go up or down it's 0 effort. If this change goes through I will just increase my characters age and change nothing else because character age has never been relevant to how I RP. She's old enough to have graduated, good enough for me tbqh.
> 
> Not a problem to adjust your age to match

And for the people who do care about their character's age and backstory and don't want a forced time skip that makes them have to retcon/redo a lot of it?

**Comment by ReturnToZender** [comment]
> > > No? The confidence issue is entirely OOC, nobody ever looks at ages except to be annoying about it. Secondly, what backstory planning? Stronger characters? It's a number you can just make go up or down it's 0 effort. If this change goes through I will just increase my characters age and change nothing else because character age has never been relevant to how I RP. She's old enough to have graduated, good enough for me tbqh.
> > 
> > 
> > Not a problem to adjust your age to match
> 
> And for the people who do care about their character's age and backstory and don't want a forced time skip that makes them have to retcon/redo a lot of it?

Forced to eat cement

**Comment by NargaInDisguise** [comment]
I'm going to be completely and utterly honest, making age a requirement changes absolutely nothing in the confidence front, its entirely decided on if I can find the character reliable and making good decisions. This arbitrary change does nothing but lock people out that otherwise are probably fine, and also why 25 and 28?

**Comment by ReturnToZender** [comment]
> I'm going to be completely and utterly honest, making age a requirement changes absolutely nothing in the confidence front, its entirely decided on if I can find the character reliable and making good decisions. This arbitrary change does nothing but lock people out that otherwise are probably fine, and also why 25 and 28?

Generally, it's nudged between 21 and 25, 24 and 30, and 25 and 28, so any of those three would be acceptable to me, this was just the one that got suggested the most

**Comment by NargaInDisguise** [comment]
Who was suggesting this? I hadn't seen any talks about this publicly, and quite frankly, if this was put to a public vote it would most certainly be shot down because its an archaic, arbitrary change that disproportionately affects a group you aren't intending to aim at.

File: modular_zubbers/code/modules/jobs/_job.dm [hunk 1]
```diff
@@ -24,7 +24,6 @@
 
 /datum/job/research_director
 	required_character_age = 21
```
Comment (lessthnthree) [inline modular_zubbers/code/modules/jobs/_job.dm:26]: So why is RD just 21?

Comment (ReturnToZender) [reply modular_zubbers/code/modules/jobs/_job.dm:26]: Leftover from when it was 21 and 25, I've been nudging it back and forth, current idea seems to be 21 is preferred so there'll be a commit for that

Comment (lessthnthree) [reply modular_zubbers/code/modules/jobs/_job.dm:26]: 21/25 seems like a good middle ground

**Comment by NargaInDisguise** [comment]
I think I am okay with 21 being a minimum age for command, but any higher gets to be arbitrary and unnecessary.

**Comment by Shroopy** [comment]
Please don't, there's no reason for this

**Comment by gummardgumphrey** [comment]
it could be a mix of 21/25/28 tbh

21 for the HoP, QM and Blueshield -- considering such jobs wouldn't require much education post-high school

25 for Captain, Head of Security, Nanotrasen Consultant, Chief Engineer, Research Director -- seems like those would require varying degrees and/or several years of job experience out of highschool

though a bit of stretch (because i agree after a certain age it can become an arbitrary), requiring an age between 26 and 28 for the CMO makes the most sense. i know "muh futuristic medical technology" and all that, but considering most doctors IRL don't exit residency until they're in their thirties, and even with regards to the advancement in technology making certain technical skills of a doctor obsolete, it'd still be generous to say the average doctor in the SS13 universe doesn't actually get to work post-residency until they're in their mid-to-late twenties, and then just add a couple years for the experience necessitated to quality for the position of CMO.

but in the broader picture, this is the type of change that will (and already has) rustled some people's feathers, and that's a good thing. one of the core flaws of all of these "SS13 furry servers" is that people are given an ungodly amount of liberty to make whatever type of character they want, and while most people have some sense of moderation, it isn't uncommon to see the occasional Mary Sue-esque character (John Shadowfang, aged 19, ex-special forces, is a capable Captain, HoS, CE, ect). this unfortunately has been the status quo for a long time, and people have gotten almost too comfortable to being able to do whatever the hell they want -- which isn't always a bad thing, because character freedom **is** good to an extent, but there needs to be certain limits to uphold people's suspension of disbelief as well as a consistent/believable roleplaying environment.

i'm in complete support of this change, and yet while i agree that certain command roles should require lower ages, and some higher ages, don't make the complete compromise of totally lowering the bar back down to an age that barely makes a dent (like minimum age 21 for every single role). you have a really good chance here to add something that these types of servers desperately need: _character believability_. as arbitrary and as minor of a chance as some may claim, it still another brick added to the foundation for an actually respectable roleplaying environment. 

if you have to upset a couple of people because their 18-20 year old captain character (HIRED BECAUSE THEIR DAD IS A NANOTRASEN LOBBYIST/THEY'RE A SUPER TALENTED AND SPECIAL TEEN, ECT) is suddenly out of a job, so be it. this still adds to the experience, even if by a small margin.

**Comment by ReturnToZender** [comment]
> it could be a mix of 21/25/28 tbh
> 
> 21 for the HoP, QM and Blueshield -- considering such jobs wouldn't require much education post-high school
> 
> 25 for Captain, Head of Security, Nanotrasen Consultant, Chief Engineer, Research Director -- seems like those would require varying degrees and/or several years of job experience out of highschool
> 
> though a bit of stretch (because i agree after a certain age it can become an arbitrary), requiring an age between 26 and 28 for the CMO makes the most sense. i know "muh futuristic medical technology" and all that, but considering most doctors IRL don't exit residency until they're in their thirties, and even with regards to the advancement in technology making certain technical skills of a doctor obsolete, it'd still be generous to say the average doctor in the SS13 universe doesn't actually get to work post-residency until they're in their mid-to-late twenties, and then just add a couple years for the experience necessitated to quality for the position of CMO.
> 
> but in the broader picture, this is the type of change that will (and already has) rustled some people's feathers, and that's a good thing. one of the core flaws of all of these "SS13 furry servers" is that people are given an ungodly amount of liberty to make whatever type of character they want, and while most people have some sense of moderation, it isn't uncommon to see the occasional Mary Sue-esque character (John Shadowfang, aged 19, ex-special forces, is a capable Captain, HoS, CE, ect). this unfortunately has been the status quo for a long time, and people have gotten almost too comfortable to being able to do whatever the hell they want -- which isn't always a bad thing, because character freedom **is** good to an extent, but there needs to be certain limits to uphold people's already shaky suspension of disbelief as well as a consistent/believable roleplaying environment.
> 
> i'm in complete support of this change, and yet while i agree that certain command roles should require lower ages, and some higher ages, don't make the complete compromise of totally lowering the bar back down to an age that barely makes a dent (like minimum age 21 for every single role). you have a really good chance here to add something that these types of servers desperately need: _character believability_. as arbitrary and as minor of a change as some may claim, this is still another brick added to the foundation for an actually respectable roleplaying environment.
> 
> if you have to upset a couple of people because their 18-20 year old captain character (HIRED BECAUSE THEIR DAD IS A NANOTRASEN LOBBYIST/THEY'RE A SUPER TALENTED AND SPECIAL TEEN, ECT) is suddenly out of a job, so be it. this still adds to the experience, even if by a small margin.

Actually if someone actually explained it by saying their dad was a Nanotrasen lobbyist it would be much better than it currently is, most people don't even do that

**Comment by NargaInDisguise** [comment]
I feel like I should try and explain a few things:
A: This doesn't affect me, I'm trying to argue from a neutral standpoint, Stryk is 25 so I can play RD/QM which are the only command roles I want to play anyways.
B: The problem of being too young for the arbitrarily chosen number only affects people that ACTUALLY CARE about their characters lore. A shitter in command isn't going to be suddenly fixed by increasing their age, they'll just increase the number with no repercussions and the cycle of bullshit continues.

**Review by Majkl-J — COMMENTED** [review]
Everyone is gonna just tune the number up and claim their character's age to be different IC. This won't result in "Interesting backstories", because frankly nobody cares about the arbitrary value, and this won't make them magically care.

For example I doubt many synths aboard are canonically actually the age they set since they left the production line (And if you managed to keep your shell pristine for 25 years, good on you).

 I have never seen it brought up aside from people telling me their backstories, and half the time their number in the funny medical console vastly differs, because it is one setting just hidden away and often forgotten about.

**Comment by ReturnToZender** [comment]
> Everyone is gonna just tune the number up and claim their character's age to be different IC. This won't result in "Interesting backstories", because frankly nobody cares about the arbitrary value, and this won't make them magically care.
> 
> For example I doubt many synths aboard are canonically actually the age they set since they left the production line (And if you managed to keep your shell pristine for 25 years, good on you).
> 
> I have never seen it brought up aside from people telling me their backstories, and half the time their number in the funny medical console vastly differs, because it is one setting just hidden away and often forgotten about.

Totally fine with this result, by the way

**Comment by projectkepler-ru** [comment]
> > Everyone is gonna just tune the number up and claim their character's age to be different IC. This won't result in "Interesting backstories", because frankly nobody cares about the arbitrary value, and this won't make them magically care.
> > For example I doubt many synths aboard are canonically actually the age they set since they left the production line (And if you managed to keep your shell pristine for 25 years, good on you).
> > I have never seen it brought up aside from people telling me their backstories, and half the time their number in the funny medical console vastly differs, because it is one setting just hidden away and often forgotten about.
> 
> Totally fine with this result, by the way


That's not a good thing

**Comment by ReturnToZender** [comment]
Closed due to stale, will re-open on request

**Pull request closed without merging**